### DISCUSSION TITLE
2.21.0版本vscode插件的代码片段功能失效

### DISCUSSION THREAD

**Discussion by shilin8805** [opensumi/core#1917] [Q&A] [answered]
2.21.0版本重构取消了VSCodeContributeRunner，以下代码执行时，snippetSuggestProvider的fromPath方法还没调用，代码片段未生成无法正常注册，导致插件的代码片段全部不生效，目前没找到合适的修改方法。
<img width="1070" alt="image" src="https://user-images.githubusercontent.com/3340210/199732157-60d08704-e259-432b-962b-dc18b4aa02c9.png">

**Comment by Aaaaash** [comment, marked as answer]
我提交了一个 pr https://github.com/opensumi/core/pull/1920 ， 应该可以修复这个问题，自动发布完 next 版本后你可以安装测试一下
